apps/backend/src/modules/users/user.repository.ts

```diff
@@ -38,6 +39,12 @@ class UserRepository implements Repository {
 		return users.map((user) => UserEntity.initialize(user));
 	}
 
+	public async findByEmail(email: string): Promise<null | UserEntity> {
```

Comment:
It's better to modify `find`method on 36 line and make it universal. For example, it takes query property, and then you use `findOne(query)`